hardhat.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     settings: {
       optimizer: {
         enabled: true,
-        runs: 1000,
+        runs: 100,
```

Comment:
I was considering implementing ClaimsManager and the Kleros-specific stuff as two separate contracts. Having to do this gave more of a reason to do that.